lib/Functional/Functional.ark

```diff
@@ -23,4 +23,16 @@
         })
         output
     }))
+
+    (let filter (fun (f L) {
```

Comment:
Just for coherence with the other functions, I think it would be better to take a list first, and then the function

Reply:
map and reduce take a function and a list, in that order (in the source), and I think these 3 functions should do this way.

Reply:
Woops, I messed up, then that's fine